lib/min_heap/min_heap.c

```diff
@@ -22,13 +22,10 @@ LOG_MODULE_REGISTER(min_heap);
  */
 static void swap(struct min_heap *heap, size_t a, size_t b)
```

Comment:
In case there is another rev of this
```suggestion
 static void swap(const struct min_heap *heap, size_t a, size_t b)
```

Reply:
Removed swap()